app/controllers/mixins/ems_common_angular.rb

```diff
@@ -356,6 +356,10 @@ def set_ems_record_vars(ems, mode = nil)
         ems.subscription    = params[:subscription] unless params[:subscription].blank?
       end
 
+      if ems.kind_of?(ManageIQ::Providers::VmwareVcd::CloudManager)
```

Comment:
@bancek is working on moving this over to `ManageIQ::Providers::Vmware::CloudManager`